doc/source/tutorial/formats.rst

```diff
@@ -31,9 +31,9 @@ specifies all entry types in a given dataset::
     11   Woldemort   Владимир      vladimir    Russian      6
     12   Woldemort   Володимир     volodimir   Ukrainian    6
     9    Harry       Harald        haralt      German       7
-    10   Harry       Harry         hæri        English      7
-    11   Harry       Гарри         gari        Russian      7
-    12   Harry       Гаррi         hari        Ukrainian    7
+    10   Harry       Harry         hæri        English      8
```

Comment:
From the dictionaries I checked they are at most what you describe as "oblique" (Harry for Henry from Heimerich; Harold from *waldan), but there is really no reason to be so detailed. I'll revert it later.

There was actually a test case failing due to my changes, so I'm enabling Travis and will start running nose locally before pushing. I'll do it later today, there are more changes to documentation.